src/WRS.jl

```diff
@@ -56,5 +57,14 @@ function pb2gen(x, y, est, q=0.5; alpha=0.05, nboot=2000)
     return est_x, est_y, est_diff, ci, pvalue, se
 end
 
+function to_be_named(x, y, est, quantiles=[0.05, 0.25, 0.5, 0.75, 0.95]; alpha=0.05, nboot=2000)
+    results = DataFrame(q=Float64[], est_x=Float64[], est_y=Float64[], est_diff=Float64[], ci_low=Float64[], ci_up=Float64[], pvalue=Float64[], se=Float64[], signif=Bool[])
+    for q in quantiles
+        est_x, est_y, est_diff, ci, pvalue, se = pb2gen(x, y, est, q; alpha=alpha, nboot=nboot)
+        # push!(results, q, est_x, est_y, est_diff, ci[1], ci[2], pvalue, se, !(ci[1]<0 && ci[2]>0))
+        push!(results, (q, est_x, est_y, est_diff, ci[1], ci[2], pvalue, se, pvalue < alpha))
```

Comment:
Are these two ways of testing for significance the same? either:
- confidence interval doesn't include zero
- p-value is less than alpha

Reply:
Yep, they should :) If my stats haven't left me ;) If we compute the confidence intervals based on alpha, i.e. our confidence level. Not sure we do that? If we always compute the 95% confidence interval, then the two will only be the same for alpha=5%.